<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>936</width>
    <height>556</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resources.qrc">
    <normaloff>:/icons/programm_icon.png</normaloff>:/icons/programm_icon.png</iconset>
  </property>
  <widget class="QWidget" name="horizontalContainer">
   <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,1">
    <item>
     <widget class="QTreeView" name="databaseList">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout" stretch="0,1,6,1">
      <item>
       <widget class="QTabWidget" name="tabWidget">
        <property name="currentIndex">
         <number>0</number>
        </property>
        <property name="usesScrollButtons">
         <bool>true</bool>
        </property>
        <property name="documentMode">
         <bool>false</bool>
        </property>
        <property name="tabsClosable">
         <bool>true</bool>
        </property>
        <property name="movable">
         <bool>true</bool>
        </property>
        <property name="tabBarAutoHide">
         <bool>false</bool>
        </property>
        <widget class="QWidget" name="add_one">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <attribute name="title">
          <string>Tab 1</string>
         </attribute>
        </widget>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="1,1,8">
        <item>
         <widget class="QPushButton" name="btnQuerySave">
          <property name="cursor">
           <cursorShape>PointingHandCursor</cursorShape>
          </property>
          <property name="toolTip">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Press &lt;span style=&quot; font-weight:600;&quot;&gt;Button&lt;/span&gt; to Save Query to File&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
          <property name="text">
           <string>&amp;Save</string>
          </property>
          <property name="icon">
           <iconset resource="resources.qrc">
            <normaloff>:/icons/speichern_schatten.png</normaloff>:/icons/speichern_schatten.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>24</width>
            <height>24</height>
           </size>
          </property>
          <property name="shortcut">
           <string>Alt+S</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btnQueryLoad">
          <property name="cursor">
           <cursorShape>PointingHandCursor</cursorShape>
          </property>
          <property name="toolTip">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Press &lt;span style=&quot; font-weight:600;&quot;&gt;Button&lt;/span&gt; to load Query from File&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
          <property name="locale">
           <locale language="English" country="UnitedKingdom"/>
          </property>
          <property name="text">
           <string>&amp;Load</string>
          </property>
          <property name="icon">
           <iconset resource="resources.qrc">
            <normaloff>:/icons/speichern_schatten.png</normaloff>:/icons/speichern_schatten.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>24</width>
            <height>24</height>
           </size>
          </property>
          <property name="shortcut">
           <string>Alt+L</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btnQueryExecute">
          <property name="cursor">
           <cursorShape>PointingHandCursor</cursorShape>
          </property>
          <property name="contextMenuPolicy">
           <enum>Qt::NoContextMenu</enum>
          </property>
          <property name="toolTip">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Press &lt;span style=&quot; font-weight:600;&quot;&gt;Button&lt;/span&gt; or &lt;span style=&quot; font-weight:600;&quot;&gt;Ctrl+Return&lt;/span&gt; to execute Query&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
          <property name="accessibleDescription">
           <string/>
          </property>
          <property name="layoutDirection">
           <enum>Qt::LeftToRight</enum>
          </property>
          <property name="locale">
           <locale language="English" country="UnitedKingdom"/>
          </property>
          <property name="inputMethodHints">
           <set>Qt::ImhNone</set>
          </property>
          <property name="text">
           <string>&amp;Execute</string>
          </property>
          <property name="icon">
           <iconset resource="resources.qrc">
            <normaloff>:/icons/ausfuehren_schatten.png</normaloff>:/icons/ausfuehren_schatten.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>24</width>
            <height>24</height>
           </size>
          </property>
          <property name="shortcut">
           <string>Alt+E, Ctrl+Return</string>
          </property>
          <property name="autoDefault">
           <bool>false</bool>
          </property>
          <property name="default">
           <bool>false</bool>
          </property>
          <property name="flat">
           <bool>false</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QTableView" name="queryResult">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="baseSize">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
        <property name="toolTip">
         <string/>
        </property>
        <property name="locale">
         <locale language="English" country="UnitedKingdom"/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QTextEdit" name="information">
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>936</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuMain_Tool_Bar">
    <property name="title">
     <string>Connections</string>
    </property>
    <widget class="QMenu" name="menuVerbinden">
     <property name="toolTip">
      <string>Select Connection and press to Connect</string>
     </property>
     <property name="title">
      <string>Connect</string>
     </property>
    </widget>
    <addaction name="actionNewConnection"/>
    <addaction name="actionEditConnection"/>
    <addaction name="menuVerbinden"/>
    <addaction name="separator"/>
    <addaction name="actionClose"/>
   </widget>
   <widget class="QMenu" name="menuHilfe">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAboutDBManager"/>
    <addaction name="actionCheckForUpdates"/>
   </widget>
   <addaction name="menuMain_Tool_Bar"/>
   <addaction name="menuHilfe"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <action name="actionNewConnection">
   <property name="text">
    <string>New</string>
   </property>
   <property name="toolTip">
    <string>Click Entry to create new Connection</string>
   </property>
  </action>
  <action name="actionEditConnection">
   <property name="text">
    <string>Manager</string>
   </property>
   <property name="toolTip">
    <string>Click Entry to edit Connections</string>
   </property>
  </action>
  <action name="actionClose">
   <property name="text">
    <string>Close</string>
   </property>
   <property name="toolTip">
    <string>Click Entry to close</string>
   </property>
  </action>
  <action name="actionSubmenu">
   <property name="text">
    <string>Submenu</string>
   </property>
  </action>
  <action name="actionAboutDBManager">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/hilfe_schatten.png</normaloff>:/icons/hilfe_schatten.png</iconset>
   </property>
   <property name="text">
    <string>about DBManager</string>
   </property>
   <property name="toolTip">
    <string>Click Entry to show Information about DBManager</string>
   </property>
  </action>
  <action name="actionCheckForUpdates">
   <property name="text">
    <string>check for Updates</string>
   </property>
   <property name="toolTip">
    <string>Click Entry to check for new updates for DBManager</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>actionClose</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>467</x>
     <y>277</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
